### PR TITLE
Add updated Manila KUTTL tests 

### PR DESCRIPTION
**Pull request by vhariria**

### CODE REVIEW

**Comment by fmount** [comment]
@vhariria @gouthampacha added [1] so we'll be able to trigger the job here when we're ready.

[1] https://github.com/openshift/release/pull/41363

**Review by gouthampacha — CHANGES_REQUESTED** [review]
Hi Vida, 

many things to fix up here.. i took a shot and got things working on your test host, could you please look at https://github.com/openstack-k8s-operators/manila-operator/pull/109 and adapt it? Or we can work on that PR instead.. another thing to note is to avoid merge commits... they mess up the diff

File: kuttl-test.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+reportFormat: JSON
+reportName: kuttl-test-manila
+namespace: openstack
+timeout: 180
+parallel: 1
+suppress:
+   - events                     # Remove spammy event logs
```
Comment (gouthampacha) [inline kuttl-test.yaml:25]: not spammy at all :)
just that we don't intend on doing string matches with the event log..

**Review by gouthampacha — CHANGES_REQUESTED** [review]
Thanks for the changes; some more pointers inline. I will close https://github.com/openstack-k8s-operators/manila-operator/pull/109/files now since you've ported the suggestions over..

File: config/samples/manila_v1beta1_manila.yaml [hunk 2]
```diff
@@ -6,19 +6,13 @@ metadata:
 spec:
   serviceUser: manila
   customServiceConfig: |
-    [DEFAULT]
+    [ DEFAULT ]
```
[the remark this answers is not in the document]
Comment (vhariria) [reply config/samples/manila_v1beta1_manila.yaml:9]: This change reverts the sample file to original file in the repo

File: config/samples/manila_v1beta1_manila.yaml [hunk 3]
```diff
@@ -6,19 +6,13 @@ metadata:
...
   secret: osp-secret
   databaseUser: manila
   rabbitMqClusterName: rabbitmq
-  manilaAPI:
-    replicas: 1
-    containerImage: quay.io/podified-antelope-centos9/openstack-manila-api:current-podified
-  manilaScheduler:
-    replicas: 1
-    containerImage: quay.io/podified-antelope-centos9/openstack-manila-scheduler:current-podified
+  manilaAPI: {}
```
[the remark this answers is not in the document]
Comment (vhariria) [reply config/samples/manila_v1beta1_manila.yaml:15]: These changes revert the file back to original repo version

**Comment by gouthampacha** [comment]
/lgtm 

Thanks for pushing this @vhariria

**Review by gouthampacha — APPROVED** [review]

**Pull request merged**